client/src/index.html

```diff
@@ -19,7 +19,7 @@
 
   <body>
     <nav class="navbar navbar-dark fixed-top bg-dark flex-md-nowrap p-0 shadow">
-      <a class="navbar-brand col-sm-3 col-md-2 mr-0" href="http://getbootstrap.com/docs/4.1/examples/dashboard/#">Company name</a>
+      <a class="navbar-brand col-sm-3 col-md-2 mr-0" href="http://getbootstrap.com/docs/4.1/examples/dashboard/#">Inventive Academy</a>
```

Comment:
remove the protocol (http:) and instead, simply use href="//getbootstrap.com..." to allow this code to work with both secure (https:) and insecure (http:) protocols. This will help avoid a "mix-content" or "insecure content" warning in the Browser.